src/components/WordAligner.js

```diff
@@ -355,6 +358,13 @@ const WordAligner = ({
     console.log('setToolSettings')
   };
 
+  useEffect(() => { // detect reset of alignments command
```

Comment:
> If we expect resetAlignments to be a state handled by the parent, then we shouldn't need a useEffect here since our component will update when our props change!

It sounds good in theory, but handleResetVerseAlignments() changes the state.  So without the useEffect we end up with repeatedly calling handleResetVerseAlignments()

Reply:
I'll look at your other suggestion about having the app handle the reset of alignments.